src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+}
+
+.menu {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-space {
+  display: flex;
+  align-items: center;
+  margin: 10px 50px 7px;
+}
+
+.menu-nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.menu-items {
+  margin: 0;
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.nav__list:last-child {
+  margin-right: 50px;
+}
+
+.nav__list-a {
+  height: 88%;
+  display: flex;
+  align-items: center;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  border-top: white 4px solid;
+  border-bottom: white 4px solid;
```

Comment:
it should not be border, in mockup it has rounded corners
make it pseudo element